pmdarima/arima/stationarity.py

```diff
@@ -172,10 +175,14 @@ def is_stationary(self, x):
         eta = (s * s).sum() / (n**2)
         s2 = (e * e).sum() / n
 
-        scalar, denom = 10, 14
+        # scalar, denom = 10, 14
+        # if self.lshort:
+        #     scalar, denom = 3, 13
```

Comment:
Is this portion being left commented out intentionally?

Reply:
Yep. Replaced by the new code:

```python
if self.lshort:
    l_ = int(np.trunc(4 * (n / 100) ** 0.25))
else:
    l_ = int(np.trunc(12 * (n / 100) ** 0.25))
```

Reply:
Left the old for posterity's sake 😉

Reply:
Gotcha. I figured, but thought I would ask!